src/Client/ClientBase.cpp

```diff
@@ -3281,6 +3290,9 @@ void ClientBase::runInteractive()
 
 bool ClientBase::processMultiQueryFromFile(const String & file_name)
 {
+    if (global_context->getApplicationType() == Context::ApplicationType::EMBEDDED_CLIENT)
```

Comment:
Let's add separate option is processing queries from file allowed into `ClientOptions``